frontend/src/pages/AllChatsPage.jsx

```diff
@@ -45,7 +85,21 @@ export default function AllChatsPage() {
     return (
         <div className="AllChatsPage FullHeight LightBackground">
             <ChatInboxHeader />
-            <ChatInbox chats={matches} users={matchedUsers} />
+            <ChatInbox chats={unblockedMatches} users={matchedUsers} />
+
+            <div className={blockedMatches.length === 0 ? 'Hidden' : ''}>
+                <div className="HiddenInbox">
+                    <h5>Hidden Conversations</h5>
+                    <button type="button" className="btn" onClick={handleInboxOpening}>
+                        {!isOpen ? (
+                            <FontAwesomeIcon icon={faChevronDown} />
+                        ) : (
+                            <FontAwesomeIcon icon={faChevronUp} />
+                        )}
+                    </button>
+                </div>
+                {isOpen && <ChatInbox chats={blockedMatches} users={matchedUsers} />}
```

Comment:
Optional: Should we add a paragraph inside the hidden section that explains something like "These chats are hidden because they include users that you have blocked."